model/testing/src/icon4py/model/testing/definitions.py

```diff
@@ -49,19 +49,11 @@ class Grid:
     description: str
     kind: GridKind
     sizes: Mapping[Literal["cell", "edge", "vertex"], int]
-    file_name: str | None = None
-    uri: str | None = None
+    file_name: str
+    uri: str
 
 
 class Grids:
```

Comment:
Are you removing this one, because it would not work with all the datatest machinery downloading files etc? I would kind of prefer to keep it here as a matter of documentation. But then again: I think the simple.py code is in the wrong place anyway. It should not be in `icon4py.model.common` but in `icon4py.model.testing`...